src/hacking/building-for-openharmony.md

```diff
@@ -0,0 +1,120 @@
+# Building for OpenHarmony 
+
+<div class="warning _note">
+Support for OpenHarmony is currently in-progress and these instructions might change from time to time and might also be incomplete.
+</div>
+
+## Get the OpenHarmony tools
+
+Building for OpenHarmony requires the following: 
+
+1. The OpenHarmony SDK. This is sufficient to compile servo as a shared library for OpenHarmony.
+2. The `hvigorw` build tool to compile apps into an app bundle and sign it.
+
+### Setting up the OpenHarmony SDK
+
+The OpenHarmony SDK is required to compile applications for OpenHarmony. 
+
+#### Downloading via DevEco Studio 
+
+[DevEco Studio] is an IDE for developing applications for HarmonyOS NEXT and OpenHarmony.
+It supports Windows and MacOS.
+You can manage installed OpenHarmony SDKs by clicking File->Settings and selecting "OpenHarmony SDK".
+After setting a suitable installation path, you can select the components you want to install for each available API version.
+DevEco Studio will automatically download and install the components for you.
+
+
+#### Manual installation of the OpenHarmony SDK (e.g. on Linux)
+
+<div class="warning _note">
+    Before rushing and downloading the OH SDK from gitee as described here, please note that you will also need `hvigor` to compile applications.
+    `hvigor` is currently only available via the [HarmonyOS NEXT commandline tools], which also contains a copy of the OpenHarmony SDK.
```

Comment:
The markdown syntax doesn't seem to have any effect inside the warning div. Both the backtick quoted text and reference links are not [rendered correctly](https://github.com/servo/book/blob/577a8c416393497d3868065c9c463dce12a85273/src/hacking/building-for-openharmony.md)

Reply:
Thanks, thats good to know!